### PR TITLE
Fixing error propagation for CosmosDB 2020-06-01-preview

### PR DESCRIPTION
**Pull request by nichatur**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
CosmosDB

**Review by kavskalyan — APPROVED** [review]

**Review by MehaKaushik — APPROVED** [review]

File: src/cosmosdb-preview/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.4.0
```
Comment (evelyn-ys) [inline src/cosmosdb-preview/HISTORY.rst:6]: Did you forget the version in `setup.py`?

Comment (nichatur) [reply src/cosmosdb-preview/HISTORY.rst:6]: I did. Thanks! :) Fixed now

**Pull request merged**